modules/frontend/src/lib/history/search-mode-badge.svelte

```diff
@@ -1,39 +1,38 @@
 <script lang="ts">
+  import Badge from '$lib/values/util/badge.svelte';
+
   interface Props {
     mode: 'match' | 'include' | 'outcome';
   }
 
   let { mode }: Props = $props();
 
-  function modeBgColor(mode: string): string {
+  // See https://www.hl7.org/fhir/R4B/valueset-search-entry-mode.html
+  function modeColor(mode: string): string {
     switch (mode) {
       case 'match':
-        return 'bg-green-100';
+        return 'green';
       case 'include':
-        return 'bg-fuchsia-100';
+        return 'fuchsia';
       default:
-        return 'bg-red-100';
+      case 'outcome':
+        return 'red';
     }
   }
 
-  function modeTextColor(mode: string): string {
+  function modeTitle(mode: string): string | undefined {
     switch (mode) {
       case 'match':
-        return 'text-green-800';
+        return 'Direct match';
       case 'include':
-        return 'text-fuchsia-800';
+        return 'Included resource';
+      case 'outcome':
```

Comment:
why do you have `outcome` here? `outcome` should also return `undefined` which is covered by `default`